.github/workflows/go.yml

```diff
@@ -10,13 +10,16 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ["1.18", "latest"]
```

Comment:
Looks like it's erroring out on "latest"

https://github.com/actions/setup-go?tab=readme-ov-file#supported-version-syntax